src/components/ui/DropDownFilter.scss

```diff
@@ -8,7 +8,7 @@ option[value="0"][disabled] {
 }
 
 .widget-drop-down-filter-loading {
-    background-color: rgba(211, 211, 211, 0.2);
+    background-color: white;
     background-image: url(./loading.gif);
     background-repeat: no-repeat;
     background-position: center center;
```

Comment:
Loading icon should display top center, else it might be out of view port on large lists
`background-position: top center;`